### DISCUSSION TITLE
Edit custom widget height

### DISCUSSION THREAD

**Discussion by hanngu** [Azure/api-management-developer-portal#1974] [General]
Hello,

Is there a way to edit the height of a custom widget iframe? It is set to 500px in sizeStylesInitial and I need it to sometimes be larger.

**Comment by malincrist** [comment]
You can edit it by clicking on "Edit widget":
![image](https://user-images.githubusercontent.com/92857141/196748301-c7447010-128a-4fe5-8fcc-489a2388a129.png)

and in the widget editor there is the "height" property which you can edit:
![image](https://user-images.githubusercontent.com/92857141/196747559-42e6b5d3-df59-4cb4-8eaa-ef70a26353f2.png)

**Reply by hanngu** [reply]
Hey, and thanks for the reply! When I create a custom React widget as described [here](https://learn.microsoft.com/en-us/azure/api-management/developer-portal-extend-custom-functionality#create-and-upload-custom-widget), my editor looks like this:
![custom widget](https://user-images.githubusercontent.com/1068830/196919730-ca44b949-26e6-4955-8d9d-559a5c8cbb20.png)

Do you have any idea what I do wrong? At this point I have not edited the original code at all.

**Reply by hanngu** [reply]
When I inspect the element I see that the elements you describe are present, but they do not show in the editor. I have tried with both Chrome and Safari.

**Reply by malincrist** [reply]
Can you please try to resize the widget editor and let us know if it works? 
![drag-widget-editor](https://user-images.githubusercontent.com/92857141/196974202-0fe99905-e658-4a1a-a3f9-6b184415176e.gif)
Looks like an issue on our side with the default size of the editor. We'll look into it. 😊

**Reply by hanngu** [reply]
Thank you! This works :D
